### ISSUE TITLE
CS107 Computer Organization & Systems  links to CS107 Programming paradigms

### ISSUE THREAD

**Issue by mrcoder16** [prakhar1989/awesome-courses#127]
Thank you for creating this great list. I believe that the CS107 "Computer Organization & Systems" Lecture Videos link to CS107 "Programming Paradigms" which is an older version of the course that contains substantially different content.

**Comment by prakhar1989** [comment]
You are right. The course website for [CS107](https://see.stanford.edu/Course/CS107/) is much different. I guess it needs to be renamed to Programming Paradigms and the link should be updated to this one.

**Comment by L1S0** [comment]
Excuse me, there are screencast of the new version of CS107, which is being taught this quarter. I think there will be more screencast as the course progress There is the link: https://web.stanford.edu/class/cs107/syllabus.html

**Comment by bklalor** [comment]
Thanks for letting us now.  I am grateful.

On Sun, Feb 4, 2018 at 12:35 AM, Lisandro Martinez <notifications@github.com

**Comment by bklalor** [comment]
*know
